api_tests/nodes/views/test_node_registrations_list.py

```diff
@@ -1,64 +1,77 @@
-# -*- coding: utf-8 -*-
-from nose.tools import *  # flake8: noqa
+import pytest
 from urlparse import urlparse
-from api.base.settings.defaults import API_BASE
 
+from api.base.settings.defaults import API_BASE
 from tests.base import ApiTestCase
 from osf_tests.factories import (
     ProjectFactory,
     RegistrationFactory,
     AuthUserFactory,
 )
 
-
```

Comment:
yes, https://stackoverflow.com/questions/33466860/expected-two-blank-lines-pep8-warning-in-python, https://www.python.org/dev/peps/pep-0008/#blank-lines doesn't